package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-data-grid",
-  "version": "0.14.46",
+  "version": "0.15.0",
```

Comment:
I see that we will do a major version bump, it is absolutely necessary, was concerned about it, good job :+1: